knotx-example/knotx-example-app/src/test/resources/test.io.knotx.KnotxServer.json

```diff
@@ -25,7 +25,7 @@
       "routing": {
         "GET": [
           {
-            "path": "/content/.*",
+            "path": ".*\\.html",
```

Comment:
I am not sure that we should limit default routing to `.*\\.html`. You can have some other URL policy which breaks this rule. Someone can set rewrite rules like this for example: [http://knotx.io/hello-rest-service/](http://knotx.io/hello-rest-service/). In my opinion last routing entry should be `.*`.